### PR TITLE
DigiDNA MCX Guest Account

### PR DESCRIPTION
**Pull request by relgit**
Added the keys on the MCX Guest Account subdomain.

### CODE REVIEW

File: Manifests/ManifestsApple/com.apple.MCX-GuestAccount.plist [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+			<string>Payload Organization</string>
+			<key>pfm_type</key>
+			<string>string</string>
+		</dict>
+		<dict>
+			<key>pfm_default</key>
+			<false/>
+			<key>pfm_description</key>
+			<string>Enables the guest account. Overrides DisableGuestAccount.</string>
+			<key>pfm_name</key>
```
Comment (kevinmcox) [inline Manifests/ManifestsApple/com.apple.MCX-GuestAccount.plist:142]: <key>pfm_documentation_url</key>
			<string>https://developer.apple.com/documentation/devicemanagement/accounts</string>
			<key>pfm_macos_min</key>
			<string>10.7</string>

File: Manifests/ManifestsApple/com.apple.MCX-GuestAccount.plist [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+							<array>
+								<true/>
+							</array>
+							<key>pfm_target</key>
+							<string>EnableGuestAccount</string>
+						</dict>
+					</array>
+				</dict>
+			</array>
+			<key>pfm_name</key>
```
Comment (kevinmcox) [inline Manifests/ManifestsApple/com.apple.MCX-GuestAccount.plist:170]: <key>pfm_documentation_url</key>
			<string>https://developer.apple.com/documentation/devicemanagement/accounts</string>
			<key>pfm_macos_min</key>
			<string>10.7</string>

**Review by kevinmcox — CHANGES_REQUESTED** [review]
Can you add in `pfm_documentation_url` and `pfm_macos_min` for each of the keys?

**Review by kevinmcox — APPROVED** [review]
Looks good, thanks!

**Review by apizz — APPROVED** [review]
It's funny to me that Apple would have created 2 different preferences that do that are phrased the opposite and yet effectively accomplish the same thing... disable or enable the guest account.

In any case, LGTM.

**Pull request merged**